.github/workflows/minimal.yml

```diff
@@ -67,6 +67,29 @@ jobs:
           command: test
           args: --features test-mock
 
+  test-net-private:
+    name: Test Suite (network-enabled tests)
```

Comment:
I think this actually needs porting some service account tokens that Travis has.

Reply:
I did add the token as a secret and this test passed a few times - but now Quay throws 429. Probably my sprint of PRs has overloaded it